external-builds/pytorch/build_pytorch_wheels/run_pytorch_tests.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -xuo pipefail
+
+cd /workspace/external-builds/pytorch/pytorch
```

Comment:
No hardcoded paths in scripts. If using bash, you can query the script's location using code like
```bash
SCRIPT_DIR="$(cd $(dirname $0) && pwd)"
```
then either `cd` or `pushd` into that directory or prefix commands with `${SCRIPT_DIR}/`